docs/nddata/index.rst

```diff
@@ -156,6 +157,57 @@ Code that uses the arithemtic methods that used to be included in
 instead subclass `~astropy.nddata.NDDataArray`; that class is equivalent to
 the original `~astropy.nddata.NDData` class.
 
+.. _nddata_transition2:
+
+Transition to astropy x.xx
+==========================
+
+The implementation of `APE 7`_ where `~astropy.nddata.NDData` underwent a major
+revision had some bugs and minor shortcomings;
+So the interface was changed in some small ways and some bugs were fixed:
+
+`~astropy.nddata.NDDataBase`:
+
++ `NDDataBase.uncertainty.getter`: is now abstract.
```

Comment:
This and the line below are causing the sphinx failures because there is no `getter` or `setter` method.